app/models/screening_list/mappable.rb

```diff
@@ -35,6 +46,7 @@ def self.included(klass)
           properties: {
             name:                    { type:     'string',
                                        analyzer: 'standard_asciifolding_nostop',
+                                       boost:    4.0,
```

Comment:
Like I mentioned, after reading [the docs](http://www.elastic.co/guide/en/elasticsearch/reference/1.x/mapping-boost-field.html) I don't think this is going to do anything. I think you're trying to tell ES to give the "name" field more weight when performing a query. When I read the docs, it seems that in order to tell it that, you need to do so via the _query_, no via the _mapping_. I could be wrong though, tell me if I am :-)

Ideally, it would be good to have a spec that demonstrates that our weighting works. For example a spec that shows that a match in the name field gets a higher score than a match in other fields. Something like:

```
# We have two docs. A query for "father xmas" returns them both. 1's score is higher than 2's:
{ id: 1, name: 'Father Xmas', alt_names: ['Santa Claus'] },
{ id: 2, name: 'Santa Claus', alt_names: ['Father Xmas'] }
```